node7/plan.sh

```diff
@@ -0,0 +1,43 @@
+pkg_name=node7
+pkg_origin=core
+pkg_version=7.5.0
+pkg_description="Node.js® is a JavaScript runtime built on Chrome's V8 JavaScript engine."
+pkg_upstream_url=https://nodejs.org/
+pkg_license=('MIT')
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_source=https://nodejs.org/dist/v${pkg_version}/node-v${pkg_version}.tar.gz
+# the archive contains a 'v' version # prefix, but the default value of
+# pkg_version is node-${pkg_version} (without the v). This tweak makes build happy
+pkg_dirname=node-v${pkg_version}
+pkg_shasum=0da8e0288b5c0f136e650b7119219968720caf88b5a67ef0591555113f0844c2
+pkg_deps=(core/glibc core/gcc-libs core/coreutils)
+pkg_build_deps=(core/python2 core/gcc core/make)
+pkg_bin_dirs=(bin)
+pkg_include_dirs=(include)
+pkg_interpreters=(bin/node)
+pkg_lib_dirs=(lib)
+
+do_prepare() {
+  # The `/usr/bin/env` path is hardcoded, so we'll add a symlink if needed.
+  # We can't do fix_interpreter here without adding a coreutils runtime dep.
```

Comment:
No, we have hardcoded `#!/usr/bin/env` in the build scripts. You get this error if you remove it ```
/hab/pkgs/core/hab-plan-build/0.17.0/20170214234630/bin/hab-plan-build: ./configure: /usr/bin/env: bad interpreter: No such file or directory
```